plugins/tests/pod/test_pod.py

```diff
@@ -19,7 +22,7 @@ def get_pod_spec():
     return pod_spec
 
 
-def test_pod_task():
+def test_pod_task_deserialization():
```

Comment:
New test that actually deserializes the jsonified `custom` dict into a `V1PodSpec` object to ensure that serialization is correct.